vg/vgimg/vgimg.go

```diff
@@ -99,12 +95,11 @@ func NewWith(o ...option) *Canvas {
 		h := c.h / vg.Inch * vg.Length(c.dpi)
 		c.img = draw.Image(image.NewRGBA(image.Rect(0, 0, int(w+0.5), int(h+0.5))))
 	}
-	if c.gc == nil {
-		h := float64(c.img.Bounds().Max.Y - c.img.Bounds().Min.Y)
-		c.gc = draw2dimg.NewGraphicContext(c.img)
-		c.gc.SetDPI(c.dpi)
-		c.gc.Scale(1, -1)
-		c.gc.Translate(0, -h)
+	if c.ctx == nil {
+		c.ctx = gg.NewContextForImage(c.img)
+		c.img = c.ctx.Image().(draw.Image)
+		// c.gc.SetDPI(c.dpi) // FIXME(sbinet) customize DPI ?
```

Comment:
What is the plan here?

Reply:
not sure.
I'll send a tentative PR against `fogleman/gg` with the ability to set and get DPI.